help([==[

Description
===========
Bioconductor provides tools for the analysis and coprehension
 of high-throughput genomic data.


More information
================
 - Homepage: https://bioconductor.org


Included extensions
===================
admisc-0.15, affxparser-1.64.0, affy-1.70.0, affycoretools-1.64.0,
affyio-1.62.0, AgiMicroRna-2.42.0, agricolae-1.3-3, ALDEx2-1.24.0, ALL-1.34.0,
annaffy-1.63.1, annotate-1.70.0, AnnotationDbi-1.54.0,
AnnotationFilter-1.16.0, AnnotationForge-1.34.0, AnnotationHub-3.0.0,
aroma.affymetrix-3.2.0, aroma.apd-0.6.0, aroma.core-3.2.2, aroma.light-3.22.0,
ash-1.0-15, ATACseqQC-1.16.0, AUCell-1.14.0, aws.s3-0.3.21,
aws.signature-0.6.0, ballgown-2.24.0, batchelor-1.8.0, baySeq-2.26.0,
beachmat-2.8.0, Biobase-2.52.0, BiocFileCache-2.0.0, BiocGenerics-0.38.0,
BiocIO-1.2.0, BiocManager-1.30.15, BiocNeighbors-1.10.0, BiocParallel-1.26.0,
BiocSingular-1.8.0, BiocStyle-2.20.0, BiocVersion-3.13.1, biomaRt-2.48.0,
biomformat-1.20.0, Biostrings-2.60.0, biovizBase-1.40.0, blme-1.0-5,
bluster-1.2.1, bookdown-0.22, BSgenome-1.60.0,
BSgenome.Cfamiliaris.UCSC.canFam3-1.4.0, BSgenome.Hsapiens.UCSC.hg19-1.4.3,
BSgenome.Hsapiens.UCSC.hg38-1.4.3, BSgenome.Mmusculus.UCSC.mm10-1.4.0,
bumphunter-1.34.0, CAMERA-1.48.0, Category-2.58.0, ccdata-1.18.0,
ccmap-1.18.0, CGHbase-1.52.0, ChIPpeakAnno-3.26.0, CNEr-1.28.0, coloc-3.2-1,
colorRamps-2.3, ComplexHeatmap-2.8.0, ConsensusClusterPlus-1.56.0,
conumee-1.26.0, crossmeta-1.18.0, cummeRbund-2.34.0, cytolib-2.4.0,
CytoML-2.4.0, dada2-1.20.0, DeconRNASeq-1.34.0, DEGseq-1.46.0,
DelayedArray-0.18.0, DelayedMatrixStats-1.14.0, derfinder-1.26.0,
derfinderHelper-1.26.0, DESeq2-1.32.0, diffcyt-1.12.0,
DirichletMultinomial-1.34.0, DNAcopy-1.66.0, docopt-0.7.1, dqrng-0.3.0,
DRIMSeq-1.20.0, DropletUtils-1.12.1, dupRadar-1.22.0, DynDoc-1.70.0,
EBImage-4.34.0, edgeR-3.34.0, egg-0.4.5, emmeans-1.6.1,
EnsDb.Hsapiens.v75-2.99.0, EnsDb.Hsapiens.v86-2.99.0, ensembldb-2.16.0,
estimability-1.3, ExperimentHub-2.0.0, extraDistr-1.9.1, fda-5.1.9,
FDb.InfiniumMethylation.hg19-2.2.0, fds-1.8, fgsea-1.18.0, filelock-1.0.2,
flowClust-3.30.0, flowCore-2.4.0, FlowSOM-2.0.0, FlowSorted.Blood.EPIC-1.10.1,
FlowSorted.CordBloodCombined.450k-1.8.0, flowStats-4.4.0, flowViz-1.56.0,
flowWorkspace-4.4.0, FRASER-1.4.0, fresh-0.2.0, gcrma-2.64.0, gdsfmt-1.28.0,
GenABEL-1.8-0, GenABEL.data-1.0.0, genefilter-1.74.0, geneLenDataBase-1.28.0,
geneplotter-1.70.0, GENESIS-2.22.0, GENIE3-1.14.0, GenomeInfoDb-1.28.0,
GenomeInfoDbData-1.2.6, GenomicAlignments-1.28.0, GenomicFeatures-1.44.0,
GenomicFiles-1.28.0, GenomicRanges-1.44.0, GenomicScores-2.4.0,
GEOquery-2.60.0, ggbio-1.40.0, ggcyto-1.20.0, ggdendro-0.1.22,
ggnewscale-0.4.5, ggpointdensity-0.1.0, ggseqlogo-0.1, ggthemes-4.2.4,
GLAD-2.56.0, Glimma-2.2.0, GlobalAncova-4.10.0, globaltest-5.46.0,
GO.db-3.13.0, goseq-1.44.0, GOstats-2.58.0, graph-1.70.0, GSEABase-1.54.0,
gsmoothr-0.1.7, GSVA-1.40.0, Gviz-1.36.1, GWASExactHW-1.01, GWASTools-1.38.0,
HDF5Array-1.20.0, hdrcde-3.4, heatmaply-1.2.1, hgu133plus2.db-3.13.0,
HMMcopy-1.34.0, Homo.sapiens-1.3.1, IHW-1.20.0,
IlluminaHumanMethylation450kanno.ilmn12.hg19-0.6.0,
IlluminaHumanMethylation450kmanifest-0.4.0,
IlluminaHumanMethylationEPICanno.ilm10b2.hg19-0.6.0,
IlluminaHumanMethylationEPICanno.ilm10b4.hg19-0.6.0,
IlluminaHumanMethylationEPICmanifest-0.3.0, illuminaio-0.34.0, impute-1.66.0,
InteractionSet-1.20.0, interactiveDisplayBase-1.30.0, IRanges-2.26.0,
isva-1.9, JASPAR2020-0.99.10, KEGG.db-3.2.4, KEGGgraph-1.52.0,
KEGGREST-1.32.0, LEA-3.4.0, limma-3.48.0, lpsymphony-1.20.0, lsa-0.73.2,
lumi-2.44.0, marray-1.70.0, maSigPro-1.64.0, MassSpecWavelet-1.58.0,
mathjaxr-1.4-0, MatrixGenerics-1.4.0, MEDIPS-1.44.0, metagenomeSeq-1.34.0,
metaMA-3.1.2, metap-1.4, metapod-1.0.0, MethylSeekR-1.32.0, methylumi-2.38.0,
Mfuzz-2.52.0, minfi-1.38.0, missMethyl-1.26.0, mixOmics-6.16.0,
MLInterfaces-1.72.0, MotifDb-1.34.0, motifmatchr-1.14.0, motifStack-1.36.0,
MsCoreUtils-1.4.0, MSnbase-2.18.0, MultiDataSet-1.20.0, multtest-2.48.0,
muscat-1.6.0, mutoss-0.1-12, mzID-1.30.0, mzR-2.26.0, NADA-1.6-1.1,
ncdfFlow-2.38.0, NMF-0.23.0, NOISeq-2.36.0, oligo-1.56.0, oligoClasses-1.54.0,
ontologyIndex-2.7, openCyto-2.4.0, org.Hs.eg.db-3.13.0, org.Mm.eg.db-3.13.0,
org.Rn.eg.db-3.13.0, OrganismDbi-1.34.0, OUTRIDER-1.10.0, pcaMethods-1.84.0,
perm-1.0-0.0, PFAM.db-3.13.0, phyloseq-1.36.0, pmp-1.4.0, polyester-1.28.0,
poweRlaw-0.70.6, preprocessCore-1.54.0, pRoloc-1.32.0, pRolocdata-1.30.0,
pRolocGUI-2.2.0, ProtGenerics-1.24.0, PRROC-1.3.1, PSCBS-0.65.0,
PureCN-1.22.1, qap-0.1-1, qlcMatrix-0.9.7, quantsmooth-1.58.0, qvalue-2.24.0,
R.devices-2.17.0, R.filesets-2.14.0, R.huge-0.9.0, rainbow-3.6,
randomcoloR-1.1.0.1, rARPACK-0.11-0, RBGL-1.68.0, RcisTarget-1.12.0,
RcppAnnoy-0.0.18, RcppHNSW-0.3.0, RcppZiggurat-0.1.6, regioneR-1.24.0,
Repitools-1.38.0, ReportingTools-2.32.0, ResidualMatrix-1.2.0,
restfulr-0.0.13, Rfast-2.0.3, rGADEM-2.40.0, Rgraphviz-2.36.0, rhdf5-2.36.0,
rhdf5filters-1.4.0, Rhdf5lib-1.14.0, Rhtslib-1.24.0, Ringo-1.56.0,
RNASeqPower-1.32.0, ROC-1.68.0, ropls-1.24.0, RProtoBufLib-2.4.0,
Rsamtools-2.8.0, Rsolnp-1.16, RSpectra-0.16-0, Rsubread-2.6.1, rsvd-1.0.5,
rtracklayer-1.52.0, S4Vectors-0.30.0, samr-3.0, SC3-1.20.0,
ScaledMatrix-1.0.0, SCANVIS-1.6.0, scater-1.20.0, scattermore-0.7,
scran-1.20.1, scrime-1.3.5, sctransform-0.3.2, scuttle-1.2.0, SeqArray-1.32.0,
seqLogo-1.58.0, SeqVarTools-1.30.0, seriation-1.2-9, Seurat-4.0.3,
SeuratObject-4.0.2, shinyBS-0.61, shinydashboardPlus-2.0.1, shinyFiles-0.9.0,
shinyhelper-0.3.2, shinypanel-0.1.4, shinyWidgets-0.6.0, ShortRead-1.50.0,
siggenes-1.66.0, Signac-1.2.1, SingleCellExperiment-1.14.1, SingleR-1.6.1,
sitmo-2.0.1, SMVar-1.3.3, SNPRelate-1.26.0, snpStats-1.42.0,
sparseMatrixStats-1.4.0, sparsesvd-0.2, SPIA-2.44.0, stageR-1.14.0,
struct-1.4.0, structToolbox-1.4.0, SummarizedExperiment-1.22.0, sva-3.40.0,
TFBSTools-1.30.0, TFMPvalue-0.0.8, tkWidgets-1.70.0, truncnorm-1.0-8,
TSP-1.1-10, TxDb.Hsapiens.UCSC.hg19.knownGene-3.2.2,
TxDb.Mmusculus.UCSC.mm10.knownGene-3.10.0, tximport-1.20.0, uwot-0.1.10,
variancePartition-1.22.0, VariantAnnotation-1.38.0, venn-1.10, vsn-3.60.0,
waiter-0.2.1, wateRmelon-1.36.0, widgetTools-1.70.0, Wrench-1.10.0,
xcms-3.14.0, XVector-0.32.0, zCompositions-1.3.4, zlibbioc-1.38.0
]==])

whatis([==[Description: Bioconductor provides tools for the analysis and coprehension
 of high-throughput genomic data.]==])
whatis([==[Homepage: https://bioconductor.org]==])
whatis([==[URL: https://bioconductor.org]==])
whatis([==[Extensions: admisc-0.15, affxparser-1.64.0, affy-1.70.0, affycoretools-1.64.0, affyio-1.62.0, AgiMicroRna-2.42.0, agricolae-1.3-3, ALDEx2-1.24.0, ALL-1.34.0, annaffy-1.63.1, annotate-1.70.0, AnnotationDbi-1.54.0, AnnotationFilter-1.16.0, AnnotationForge-1.34.0, AnnotationHub-3.0.0, aroma.affymetrix-3.2.0, aroma.apd-0.6.0, aroma.core-3.2.2, aroma.light-3.22.0, ash-1.0-15, ATACseqQC-1.16.0, AUCell-1.14.0, aws.s3-0.3.21, aws.signature-0.6.0, ballgown-2.24.0, batchelor-1.8.0, baySeq-2.26.0, beachmat-2.8.0, Biobase-2.52.0, BiocFileCache-2.0.0, BiocGenerics-0.38.0, BiocIO-1.2.0, BiocManager-1.30.15, BiocNeighbors-1.10.0, BiocParallel-1.26.0, BiocSingular-1.8.0, BiocStyle-2.20.0, BiocVersion-3.13.1, biomaRt-2.48.0, biomformat-1.20.0, Biostrings-2.60.0, biovizBase-1.40.0, blme-1.0-5, bluster-1.2.1, bookdown-0.22, BSgenome-1.60.0, BSgenome.Cfamiliaris.UCSC.canFam3-1.4.0, BSgenome.Hsapiens.UCSC.hg19-1.4.3, BSgenome.Hsapiens.UCSC.hg38-1.4.3, BSgenome.Mmusculus.UCSC.mm10-1.4.0, bumphunter-1.34.0, CAMERA-1.48.0, Category-2.58.0, ccdata-1.18.0, ccmap-1.18.0, CGHbase-1.52.0, ChIPpeakAnno-3.26.0, CNEr-1.28.0, coloc-3.2-1, colorRamps-2.3, ComplexHeatmap-2.8.0, ConsensusClusterPlus-1.56.0, conumee-1.26.0, crossmeta-1.18.0, cummeRbund-2.34.0, cytolib-2.4.0, CytoML-2.4.0, dada2-1.20.0, DeconRNASeq-1.34.0, DEGseq-1.46.0, DelayedArray-0.18.0, DelayedMatrixStats-1.14.0, derfinder-1.26.0, derfinderHelper-1.26.0, DESeq2-1.32.0, diffcyt-1.12.0, DirichletMultinomial-1.34.0, DNAcopy-1.66.0, docopt-0.7.1, dqrng-0.3.0, DRIMSeq-1.20.0, DropletUtils-1.12.1, dupRadar-1.22.0, DynDoc-1.70.0, EBImage-4.34.0, edgeR-3.34.0, egg-0.4.5, emmeans-1.6.1, EnsDb.Hsapiens.v75-2.99.0, EnsDb.Hsapiens.v86-2.99.0, ensembldb-2.16.0, estimability-1.3, ExperimentHub-2.0.0, extraDistr-1.9.1, fda-5.1.9, FDb.InfiniumMethylation.hg19-2.2.0, fds-1.8, fgsea-1.18.0, filelock-1.0.2, flowClust-3.30.0, flowCore-2.4.0, FlowSOM-2.0.0, FlowSorted.Blood.EPIC-1.10.1, FlowSorted.CordBloodCombined.450k-1.8.0, flowStats-4.4.0, flowViz-1.56.0, flowWorkspace-4.4.0, FRASER-1.4.0, fresh-0.2.0, gcrma-2.64.0, gdsfmt-1.28.0, GenABEL-1.8-0, GenABEL.data-1.0.0, genefilter-1.74.0, geneLenDataBase-1.28.0, geneplotter-1.70.0, GENESIS-2.22.0, GENIE3-1.14.0, GenomeInfoDb-1.28.0, GenomeInfoDbData-1.2.6, GenomicAlignments-1.28.0, GenomicFeatures-1.44.0, GenomicFiles-1.28.0, GenomicRanges-1.44.0, GenomicScores-2.4.0, GEOquery-2.60.0, ggbio-1.40.0, ggcyto-1.20.0, ggdendro-0.1.22, ggnewscale-0.4.5, ggpointdensity-0.1.0, ggseqlogo-0.1, ggthemes-4.2.4, GLAD-2.56.0, Glimma-2.2.0, GlobalAncova-4.10.0, globaltest-5.46.0, GO.db-3.13.0, goseq-1.44.0, GOstats-2.58.0, graph-1.70.0, GSEABase-1.54.0, gsmoothr-0.1.7, GSVA-1.40.0, Gviz-1.36.1, GWASExactHW-1.01, GWASTools-1.38.0, HDF5Array-1.20.0, hdrcde-3.4, heatmaply-1.2.1, hgu133plus2.db-3.13.0, HMMcopy-1.34.0, Homo.sapiens-1.3.1, IHW-1.20.0, IlluminaHumanMethylation450kanno.ilmn12.hg19-0.6.0, IlluminaHumanMethylation450kmanifest-0.4.0, IlluminaHumanMethylationEPICanno.ilm10b2.hg19-0.6.0, IlluminaHumanMethylationEPICanno.ilm10b4.hg19-0.6.0, IlluminaHumanMethylationEPICmanifest-0.3.0, illuminaio-0.34.0, impute-1.66.0, InteractionSet-1.20.0, interactiveDisplayBase-1.30.0, IRanges-2.26.0, isva-1.9, JASPAR2020-0.99.10, KEGG.db-3.2.4, KEGGgraph-1.52.0, KEGGREST-1.32.0, LEA-3.4.0, limma-3.48.0, lpsymphony-1.20.0, lsa-0.73.2, lumi-2.44.0, marray-1.70.0, maSigPro-1.64.0, MassSpecWavelet-1.58.0, mathjaxr-1.4-0, MatrixGenerics-1.4.0, MEDIPS-1.44.0, metagenomeSeq-1.34.0, metaMA-3.1.2, metap-1.4, metapod-1.0.0, MethylSeekR-1.32.0, methylumi-2.38.0, Mfuzz-2.52.0, minfi-1.38.0, missMethyl-1.26.0, mixOmics-6.16.0, MLInterfaces-1.72.0, MotifDb-1.34.0, motifmatchr-1.14.0, motifStack-1.36.0, MsCoreUtils-1.4.0, MSnbase-2.18.0, MultiDataSet-1.20.0, multtest-2.48.0, muscat-1.6.0, mutoss-0.1-12, mzID-1.30.0, mzR-2.26.0, NADA-1.6-1.1, ncdfFlow-2.38.0, NMF-0.23.0, NOISeq-2.36.0, oligo-1.56.0, oligoClasses-1.54.0, ontologyIndex-2.7, openCyto-2.4.0, org.Hs.eg.db-3.13.0, org.Mm.eg.db-3.13.0, org.Rn.eg.db-3.13.0, OrganismDbi-1.34.0, OUTRIDER-1.10.0, pcaMethods-1.84.0, perm-1.0-0.0, PFAM.db-3.13.0, phyloseq-1.36.0, pmp-1.4.0, polyester-1.28.0, poweRlaw-0.70.6, preprocessCore-1.54.0, pRoloc-1.32.0, pRolocdata-1.30.0, pRolocGUI-2.2.0, ProtGenerics-1.24.0, PRROC-1.3.1, PSCBS-0.65.0, PureCN-1.22.1, qap-0.1-1, qlcMatrix-0.9.7, quantsmooth-1.58.0, qvalue-2.24.0, R.devices-2.17.0, R.filesets-2.14.0, R.huge-0.9.0, rainbow-3.6, randomcoloR-1.1.0.1, rARPACK-0.11-0, RBGL-1.68.0, RcisTarget-1.12.0, RcppAnnoy-0.0.18, RcppHNSW-0.3.0, RcppZiggurat-0.1.6, regioneR-1.24.0, Repitools-1.38.0, ReportingTools-2.32.0, ResidualMatrix-1.2.0, restfulr-0.0.13, Rfast-2.0.3, rGADEM-2.40.0, Rgraphviz-2.36.0, rhdf5-2.36.0, rhdf5filters-1.4.0, Rhdf5lib-1.14.0, Rhtslib-1.24.0, Ringo-1.56.0, RNASeqPower-1.32.0, ROC-1.68.0, ropls-1.24.0, RProtoBufLib-2.4.0, Rsamtools-2.8.0, Rsolnp-1.16, RSpectra-0.16-0, Rsubread-2.6.1, rsvd-1.0.5, rtracklayer-1.52.0, S4Vectors-0.30.0, samr-3.0, SC3-1.20.0, ScaledMatrix-1.0.0, SCANVIS-1.6.0, scater-1.20.0, scattermore-0.7, scran-1.20.1, scrime-1.3.5, sctransform-0.3.2, scuttle-1.2.0, SeqArray-1.32.0, seqLogo-1.58.0, SeqVarTools-1.30.0, seriation-1.2-9, Seurat-4.0.3, SeuratObject-4.0.2, shinyBS-0.61, shinydashboardPlus-2.0.1, shinyFiles-0.9.0, shinyhelper-0.3.2, shinypanel-0.1.4, shinyWidgets-0.6.0, ShortRead-1.50.0, siggenes-1.66.0, Signac-1.2.1, SingleCellExperiment-1.14.1, SingleR-1.6.1, sitmo-2.0.1, SMVar-1.3.3, SNPRelate-1.26.0, snpStats-1.42.0, sparseMatrixStats-1.4.0, sparsesvd-0.2, SPIA-2.44.0, stageR-1.14.0, struct-1.4.0, structToolbox-1.4.0, SummarizedExperiment-1.22.0, sva-3.40.0, TFBSTools-1.30.0, TFMPvalue-0.0.8, tkWidgets-1.70.0, truncnorm-1.0-8, TSP-1.1-10, TxDb.Hsapiens.UCSC.hg19.knownGene-3.2.2, TxDb.Mmusculus.UCSC.mm10.knownGene-3.10.0, tximport-1.20.0, uwot-0.1.10, variancePartition-1.22.0, VariantAnnotation-1.38.0, venn-1.10, vsn-3.60.0, waiter-0.2.1, wateRmelon-1.36.0, widgetTools-1.70.0, Wrench-1.10.0, xcms-3.14.0, XVector-0.32.0, zCompositions-1.3.4, zlibbioc-1.38.0]==])

local root = "/home/software/EasyBuild/software/R-bundle-Bioconductor/3.13-foss-2021a-R-4.1.0"

conflict("R-bundle-Bioconductor")

if not ( isloaded("FlexiBLAS/3.0.4") ) then
    load("FlexiBLAS/3.0.4")
end

if not ( isloaded("FFTW/3.3.9") ) then
    load("FFTW/3.3.9")
end

if not ( isloaded("ScaLAPACK/2.1.0-fb") ) then
    load("ScaLAPACK/2.1.0-fb")
end

if not ( isloaded("R/4.1.0") ) then
    load("R/4.1.0")
end

if not ( isloaded("Boost/1.76.0") ) then
    load("Boost/1.76.0")
end

if not ( isloaded("GSL/2.7") ) then
    load("GSL/2.7")
end

if not ( isloaded("HDF5/1.10.7") ) then
    load("HDF5/1.10.7")
end

if not ( isloaded("ncdf4/1.17-R-4.1.0") ) then
    load("ncdf4/1.17-R-4.1.0")
end

if not ( isloaded("arrow-R/6.0.0.2-R-4.1.0") ) then
    load("arrow-R/6.0.0.2-R-4.1.0")
end

prepend_path("CMAKE_PREFIX_PATH", root)
setenv("EBROOTRMINBUNDLEMINBIOCONDUCTOR", root)
setenv("EBVERSIONRMINBUNDLEMINBIOCONDUCTOR", "3.13")
setenv("EBDEVELRMINBUNDLEMINBIOCONDUCTOR", pathJoin(root, "easybuild/MPI-GCC-10.3.0-OpenMPI-4.1.1-R-bundle-Bioconductor-3.13-R-4.1.0-easybuild-devel"))

prepend_path("R_LIBS_SITE", root)
-- Built with EasyBuild version 4.6.0
setenv("EBEXTSLISTRMINBUNDLEMINBIOCONDUCTOR", "BiocGenerics-0.38.0,Biobase-2.52.0,S4Vectors-0.30.0,IRanges-2.26.0,GenomeInfoDbData-1.2.6,GenomeInfoDb-1.28.0,zlibbioc-1.38.0,XVector-0.32.0,Biostrings-2.60.0,KEGGREST-1.32.0,AnnotationDbi-1.54.0,GenomicRanges-1.44.0,BiocParallel-1.26.0,Rhtslib-1.24.0,Rsamtools-2.8.0,MatrixGenerics-1.4.0,DelayedArray-0.18.0,SummarizedExperiment-1.22.0,GenomicAlignments-1.28.0,ShortRead-1.50.0,graph-1.70.0,affyio-1.62.0,preprocessCore-1.54.0,BiocManager-1.30.15,affy-1.70.0,GO.db-3.13.0,limma-3.48.0,RBGL-1.68.0,org.Hs.eg.db-3.13.0,AnnotationForge-1.34.0,KEGG.db-3.2.4,annaffy-1.63.1,gcrma-2.64.0,oligoClasses-1.54.0,edgeR-3.34.0,PFAM.db-3.13.0,perm-1.0-0.0,baySeq-2.26.0,qvalue-2.24.0,impute-1.66.0,shinyFiles-0.9.0,samr-3.0,DEGseq-1.46.0,hgu133plus2.db-3.13.0,illuminaio-0.34.0,BiocIO-1.2.0,restfulr-0.0.13,rtracklayer-1.52.0,filelock-1.0.2,BiocFileCache-2.0.0,biomaRt-2.48.0,GenomicFeatures-1.44.0,bumphunter-1.34.0,multtest-2.48.0,scrime-1.3.5,siggenes-1.66.0,DynDoc-1.70.0,NOISeq-2.36.0,Rgraphviz-2.36.0,RNASeqPower-1.32.0,annotate-1.70.0,GSEABase-1.54.0,genefilter-1.74.0,Category-2.58.0,GOstats-2.58.0,BSgenome-1.60.0,VariantAnnotation-1.38.0,interactiveDisplayBase-1.30.0,BiocVersion-3.13.1,AnnotationHub-3.0.0,AnnotationFilter-1.16.0,ProtGenerics-1.24.0,ensembldb-2.16.0,biovizBase-1.40.0,OrganismDbi-1.34.0,ggbio-1.40.0,geneplotter-1.70.0,DESeq2-1.32.0,ReportingTools-2.32.0,Glimma-2.2.0,affycoretools-1.64.0,TxDb.Hsapiens.UCSC.hg19.knownGene-3.2.2,Homo.sapiens-1.3.1,BSgenome.Hsapiens.UCSC.hg19-1.4.3,AgiMicroRna-2.42.0,geneLenDataBase-1.28.0,goseq-1.44.0,KEGGgraph-1.52.0,GEOquery-2.60.0,RSpectra-0.16-0,rARPACK-0.11-0,mixOmics-6.16.0,Rhdf5lib-1.14.0,rhdf5filters-1.4.0,rhdf5-2.36.0,HDF5Array-1.20.0,sparseMatrixStats-1.4.0,DelayedMatrixStats-1.14.0,minfi-1.38.0,FDb.InfiniumMethylation.hg19-2.2.0,methylumi-2.38.0,lumi-2.44.0,widgetTools-1.70.0,tkWidgets-1.70.0,Mfuzz-2.52.0,admisc-0.15,venn-1.10,maSigPro-1.64.0,SPIA-2.44.0,Gviz-1.36.1,cummeRbund-2.34.0,GenomicFiles-1.28.0,derfinderHelper-1.26.0,derfinder-1.26.0,polyester-1.28.0,Rsubread-2.6.1,pcaMethods-1.84.0,marray-1.70.0,CGHbase-1.52.0,Wrench-1.10.0,lpsymphony-1.20.0,IHW-1.20.0,metagenomeSeq-1.34.0,gdsfmt-1.28.0,SNPRelate-1.26.0,biomformat-1.20.0,phyloseq-1.36.0,NADA-1.6-1.1,truncnorm-1.0-8,zCompositions-1.3.4,RcppZiggurat-0.1.6,Rfast-2.0.3,ALDEx2-1.24.0,dada2-1.20.0,LEA-3.4.0,tximport-1.20.0,SingleCellExperiment-1.14.1,beachmat-2.8.0,RcppAnnoy-0.0.18,RcppHNSW-0.3.0,BiocNeighbors-1.10.0,rsvd-1.0.5,ScaledMatrix-1.0.0,BiocSingular-1.8.0,scuttle-1.2.0,scater-1.20.0,sitmo-2.0.1,dqrng-0.3.0,bluster-1.2.1,metapod-1.0.0,scran-1.20.1,SC3-1.20.0,ComplexHeatmap-2.8.0,GENIE3-1.14.0,dupRadar-1.22.0,DNAcopy-1.66.0,sva-3.40.0,ballgown-2.24.0,DropletUtils-1.12.1,DeconRNASeq-1.34.0,GSVA-1.40.0,PureCN-1.22.1,globaltest-5.46.0,GlobalAncova-4.10.0,vsn-3.60.0,mzID-1.30.0,mzR-2.26.0,MsCoreUtils-1.4.0,MSnbase-2.18.0,MassSpecWavelet-1.58.0,xcms-3.14.0,CAMERA-1.48.0,fgsea-1.18.0,GWASExactHW-1.01,quantsmooth-1.58.0,GWASTools-1.38.0,SeqArray-1.32.0,SeqVarTools-1.30.0,GENESIS-2.22.0,MLInterfaces-1.72.0,pRoloc-1.32.0,pRolocdata-1.30.0,fresh-0.2.0,waiter-0.2.1,shinydashboardPlus-2.0.1,shinyhelper-0.3.2,shinyWidgets-0.6.0,pRolocGUI-2.2.0,EBImage-4.34.0,GenomicScores-2.4.0,BSgenome.Mmusculus.UCSC.mm10-1.4.0,TxDb.Mmusculus.UCSC.mm10.knownGene-3.10.0,regioneR-1.24.0,InteractionSet-1.20.0,ChIPpeakAnno-3.26.0,seqLogo-1.58.0,rGADEM-2.40.0,MotifDb-1.34.0,motifStack-1.36.0,ATACseqQC-1.16.0,ResidualMatrix-1.2.0,batchelor-1.8.0,gsmoothr-0.1.7,Ringo-1.56.0,R.devices-2.17.0,R.filesets-2.14.0,aroma.light-3.22.0,PSCBS-0.65.0,aroma.core-3.2.2,R.huge-0.9.0,aroma.apd-0.6.0,aroma.affymetrix-3.2.0,Rsolnp-1.16,Repitools-1.38.0,BSgenome.Hsapiens.UCSC.hg38-1.4.3,MEDIPS-1.44.0,RProtoBufLib-2.4.0,cytolib-2.4.0,flowCore-2.4.0,mutoss-0.1-12,mathjaxr-1.4-0,metap-1.4,uwot-0.1.10,scattermore-0.7,sctransform-0.3.2,SeuratObject-4.0.2,Seurat-4.0.3,ALL-1.34.0,ConsensusClusterPlus-1.56.0,flowViz-1.56.0,ncdfFlow-2.38.0,aws.signature-0.6.0,aws.s3-0.3.21,flowWorkspace-4.4.0,ash-1.0-15,hdrcde-3.4,rainbow-3.6,fds-1.8,fda-5.1.9,flowStats-4.4.0,flowClust-3.30.0,openCyto-2.4.0,ggcyto-1.20.0,CytoML-2.4.0,colorRamps-2.3,ggnewscale-0.4.5,ggpointdensity-0.1.0,FlowSOM-2.0.0,HMMcopy-1.34.0,diffcyt-1.12.0,blme-1.0-5,variancePartition-1.22.0,muscat-1.6.0,IlluminaHumanMethylation450kmanifest-0.4.0,IlluminaHumanMethylationEPICmanifest-0.3.0,IlluminaHumanMethylation450kanno.ilmn12.hg19-0.6.0,IlluminaHumanMethylationEPICanno.ilm10b2.hg19-0.6.0,IlluminaHumanMethylationEPICanno.ilm10b4.hg19-0.6.0,conumee-1.26.0,BSgenome.Cfamiliaris.UCSC.canFam3-1.4.0,ExperimentHub-2.0.0,SingleR-1.6.1,FlowSorted.Blood.EPIC-1.10.1,FlowSorted.CordBloodCombined.450k-1.8.0,DRIMSeq-1.20.0,stageR-1.14.0,isva-1.9,org.Mm.eg.db-3.13.0,org.Rn.eg.db-3.13.0,ROC-1.68.0,wateRmelon-1.36.0,GLAD-2.56.0,missMethyl-1.26.0,MethylSeekR-1.32.0,affxparser-1.64.0,ccdata-1.18.0,lsa-0.73.2,ccmap-1.18.0,oligo-1.56.0,SMVar-1.3.3,metaMA-3.1.2,randomcoloR-1.1.0.1,shinyBS-0.61,shinypanel-0.1.4,crossmeta-1.18.0,snpStats-1.42.0,coloc-3.2-1,SCANVIS-1.6.0,EnsDb.Hsapiens.v86-2.99.0,agricolae-1.3-3,bookdown-0.22,BiocStyle-2.20.0,estimability-1.3,emmeans-1.6.1,ggdendro-0.1.22,pmp-1.4.0,MultiDataSet-1.20.0,ropls-1.24.0,ontologyIndex-2.7,struct-1.4.0,ggthemes-4.2.4,structToolbox-1.4.0,EnsDb.Hsapiens.v75-2.99.0,ggseqlogo-0.1,sparsesvd-0.2,docopt-0.7.1,qlcMatrix-0.9.7,Signac-1.2.1,poweRlaw-0.70.6,CNEr-1.28.0,DirichletMultinomial-1.34.0,TFMPvalue-0.0.8,TFBSTools-1.30.0,motifmatchr-1.14.0,extraDistr-1.9.1,PRROC-1.3.1,TSP-1.1-10,qap-0.1-1,seriation-1.2-9,egg-0.4.5,heatmaply-1.2.1,OUTRIDER-1.10.0,FRASER-1.4.0,GenABEL.data-1.0.0,GenABEL-1.8-0,JASPAR2020-0.99.10,AUCell-1.14.0,RcisTarget-1.12.0,NMF-0.23.0")
